<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDAMainWindow</class>
 <widget class="QMainWindow" name="QDAMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>1118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="basicLebel">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 127);</string>
        </property>
        <property name="text">
         <string>基础功能</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="envCheckButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>110</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>检查环境</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="envCheckLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>110</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
</string>
          </property>
          <property name="lineWidth">
           <number>5</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string>检查开发环境</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="projGenLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>110</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
</string>
          </property>
          <property name="lineWidth">
           <number>5</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string>基于CMake的C/C++跨平台框架生成，默认支持Windows、MacOS、Linux(Ubuntu、Debian)、Android、iOS等平台的编译，工程使用C++版Qt开发。</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="projGenButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>110</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>跨平台C/C++

工程框架生成</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pkgGenButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>110</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>多平台

安装包制作</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="pkgGenLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>110</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>编译产物的安装包制作功能，支持提供多个平台的打包脚本模板，能满足基本打包需求，现阶段支持Windows平台(基于nsis打包exe安装包)、Linux平台(rpm安装包制作)，用户可以自定义修改。</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="extendLebel">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 127);</string>
        </property>
        <property name="text">
         <string>扩展功能</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QLabel" name="dataEncryptLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>110</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>规划中</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="dataDecryptLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>110</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>规划中</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="dataDecryptButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>110</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>数据解密</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="dataEncryptButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>110</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>数据加密</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1319</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="projectMenu">
    <property name="title">
     <string>Project(项目)</string>
    </property>
    <addaction name="actionEnvCheck"/>
    <addaction name="actionProjectNew"/>
    <addaction name="actionProjectOpen"/>
    <addaction name="actionProjectReview"/>
   </widget>
   <widget class="QMenu" name="exampleMenu">
    <property name="title">
     <string>Example(示例工程)</string>
    </property>
    <addaction name="actionExampleView"/>
   </widget>
   <widget class="QMenu" name="planMenu">
    <property name="title">
     <string>Plan(计划)</string>
    </property>
    <addaction name="actionPlan"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help(帮助)</string>
    </property>
    <addaction name="actionWelcome"/>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="projectMenu"/>
   <addaction name="exampleMenu"/>
   <addaction name="planMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVersion">
   <property name="text">
    <string>Version(版本)</string>
   </property>
  </action>
  <action name="actionEnvCheck">
   <property name="text">
    <string>Check Env(检查环境)</string>
   </property>
  </action>
  <action name="actionProjectNew">
   <property name="text">
    <string>New Project(新建项目)</string>
   </property>
  </action>
  <action name="actionExampleReview">
   <property name="text">
    <string>Preview(示例预览)</string>
   </property>
  </action>
  <action name="actionEncrypt">
   <property name="text">
    <string>Encrypt(数据加密)</string>
   </property>
  </action>
  <action name="actionDecrypt">
   <property name="text">
    <string>Decrypt(数据解密)</string>
   </property>
  </action>
  <action name="actionPlan">
   <property name="text">
    <string>Plan(计划)</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Custom(自定义)</string>
   </property>
  </action>
  <action name="actionProjectOpen">
   <property name="text">
    <string>Open Project(打开项目)</string>
   </property>
  </action>
  <action name="actionProjectReview">
   <property name="text">
    <string>View Project(预览项目)</string>
   </property>
  </action>
  <action name="actionModuleAdd">
   <property name="text">
    <string>Add Module(添加模块)</string>
   </property>
  </action>
  <action name="actionClassAdd">
   <property name="text">
    <string>Add Class(添加类)</string>
   </property>
  </action>
  <action name="actionExampleView">
   <property name="text">
    <string>Preview(示例预览)</string>
   </property>
  </action>
  <action name="actionWelcome">
   <property name="text">
    <string>Welcome(主页)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
